appeals/e2e/cypress/page_objects/caseDetailsPage.js

```diff
@@ -195,12 +203,7 @@ export class CaseDetailsPage extends Page {
 	}
 
 	verifyChangeStartDate() {
-		const dateToday = new Date(); // Note: Months are 0-indexed, so 10 is November
-		const formattedDate = new Intl.DateTimeFormat('en-GB', {
-			day: 'numeric',
-			month: 'long',
-			year: 'numeric'
-		}).format(dateToday); // Format the date
+		const formattedDate = dateTimeSection.formatDate();
 		cy.get('.appeal-start-date > .govuk-summary-list__value')
```

Comment:
I'd suggest pulling this selector out into the `elements` collection on line 35 of this file